src/Http/Api/Contracts/HasModel.php

```diff
@@ -78,7 +82,7 @@ public function getPostmanModel(): string
     protected function unguardIfNeeded()
     {
         if ($this->unguard) {
-            self::$model->unguard();
+            $this->getModel()::unguard();
```

Comment:
Is static call correct? Previous was not static? Based on existing call this should be:

```
$this->getModel()->unguard();
```

Reply:
It does look like it should be static, but curious as to why it was using `->` previously.

Reply:
This should have always been static so would suggest keeping new call